.github/workflows/build-publish-develop-pr.yml

```
@@ -0,0 +1,76 @@
+name: "Build and Publish Chainlink"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - develop
+
+jobs:
+  goreleaser-build-publish-chainlink:
+    runs-on:
+      labels: ubuntu22.04-16cores-64GB
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633 # v4.1.2
+
+      - name: Get image tag
+        id: get-image-tag
+        run: |
+          short_sha=$(git rev-parse --short HEAD)
+          if [[ ${{ github.event_name }} == 'push' ]]; then
+            echo "image-tag=develop-${short_sha}" >> $GITHUB_OUTPUT
+          else
+            if [[ ${{ github.head_ref }} == 'release/'* ]]; then
+              echo "image-tag=release-${short_sha}" >> $GITHUB_OUTPUT
+            else
+              echo "image-tag=pr-${{ github.event.number }}-${short_sha}" >> $GITHUB_OUTPUT
+            fi
+          fi
```

```suggestion
          if [[ ${{ github.event_name }} == 'push' ]]; then
            echo "image-tag=develop-${short_sha}" | tee -a $GITHUB_OUTPUT
          else
            if [[ ${{ github.head_ref }} == 'release/'* ]]; then
              echo "image-tag=release-${short_sha}" | tee -a $GITHUB_OUTPUT
            else
              echo "image-tag=pr-${{ github.event.number }}-${short_sha}" | tee -a $GITHUB_OUTPUT
            fi
          fi
```

Not necessary but I tend to redirect to GITHUB_OUTPUT with `tee` because it will print it to stdout too (for debugging/visbility).